projects/cdk/services/destroy.service.ts

```diff
@@ -2,7 +2,12 @@ import {Injectable, OnDestroy} from '@angular/core';
 import {ReplaySubject} from 'rxjs';
 
 /**
+ * @note:
  * Observable abstraction over ngOnDestroy to use with takeUntil
+ *
+ * Why we use `ReplaySubject` instead of `Subject`?
+ * Well, we’ll use ReplaySubject to emit the last message in case
+ * the subscription is ended after the component is destroyed.
```

Comment:
nitpicking:
You can duplicate it here

https://taiga-ui.dev/services/destroy-service